pkgs/top-level/python-packages.nix

```diff
@@ -4068,6 +4074,8 @@ in {
 
   libtorrent-rasterbar = (toPythonModule (pkgs.libtorrent-rasterbar.override { inherit python; })).python;
 
+  libtorrent-rasterbar-1_2_x = (toPythonModule (pkgs.libtorrent-rasterbar-1_2_x.override { inherit python; })).python;
```

Comment:
Please use a pythonOverride instead.

Reply:
It's a different libtorrent lib, not a different python.

Reply:
I don't think we need this in pythonPackages. We can create it locally in the final program. Same applies to the other new packages.

Reply:
Ah. Is there any example of this procedure?

Reply:
https://github.com/NixOS/nixpkgs/blob/a07c903e121ff73e11637aa479b9f7ce7ce32cf6/pkgs/applications/misc/privacyidea/default.nix#L8

Reply:
ok I think I did that well with the libtorrent but I don't know how to do the same for the others.